index.d/caddyserver.yml

```diff
@@ -6,6 +6,8 @@ Projects:
     git-branch: master
     git-path: caddy_server/centos7
     target-file: Dockerfile
+    prebuild-script: caddy_server/centos7/cccp-prebuild-sh
```

Comment:
@mohammedzee1000 should it be `prebuild-script: cccp-prebuild-sh` instead. because context goes to that directory already. 
please check this https://github.com/CentOS/container-pipeline-service/blob/master/pre_build/pre-build-job.yml#L62

Reply:
ah thanks